install.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+virtualenv --quiet --python=/usr/bin/python2.7 scancode-server
```

Comment:
Where do you get virtualenv from?
Could you try to see how the scancode-toolkit configure script works and use this instead?

Reply:
Okk!! @pombredanne i will see to how that works! but plz let me know the problem this code. Like it does what we want it to.

Reply:
This script is fine. But I would rather have a proper ./configure, scancode-toolkit style

Reply:
ok!! will work on that!